package.json

```diff
@@ -34,11 +34,13 @@
     "express": "^4.17.1",
     "file-loader": "^6.2.0",
     "jsonwebtoken": "^8.5.1",
+    "mysql2": "^2.2.5",
```

Comment:
If you are using mysql, then you can remove `pg` and `sqlite3` from your dependencies.